.terraform-version

```diff
@@ -1 +0,0 @@
-0.11.14
```

Comment:
Would it be better to update this to `0.12.12` or `0.12` or is the thinking that we shouldn't preference a version for `tfenv` in our repos?

Reply:
As we've migrated some of the other modules to TF 12, we've added a versions.tf file (also in this PR) which constrains the major version. I think that makes sense as a more tool-agnostic approach. `tfenv` and specific `.terraform-version` files can still be managed at the project level.

Reply:
That's fair. I've only found it useful when using `terraform 0.12upgrade` in this repo. But generally I do work from other repos. So I guess its up to you.